modules/exploits/windows/http/kentico_staging_syncserver.rb

```diff
@@ -0,0 +1,102 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Kentico CMS Staging SyncServer Unserialize Remote Command Execution',
+      'Description' => %q{
+        This module exploits a vulnerability in the Kentico CMS platform versions 12.0.14 and earlier.
+        Remote Command Execution is possible via unauthenticated XML requests to the Staging Service
+        SyncServer.asmx interface ProcessSynchronizationTaskData method stagingTaskData parameter. XML
+        input is passed to an insecure .NET deserialize call which allows for remote command execution.
+      },
+      'DisclosureDate' => '2019-04-15',
+      'Author' =>
+        [
+          'Manoj Cherukuri',  # Discovery
+          'Justin LeMay',     # Discovery
+          'aushack',          # msf exploit
+        ],
+      'References' =>
+        [
+          ['CVE', '2019-10068'],
+          ['URL', 'https://www.aon.com/cyber-solutions/aon_cyber_labs/unauthenticated-remote-code-execution-in-kentico-cms/']
+        ],
+      'License' => MSF_LICENSE,
+      'Platform' => 'win',
+      'Arch' => ARCH_CMD,
+      'DefaultTarget' => 0,
+      'Stance' => Msf::Exploit::Stance::Aggressive,
```

Comment:
I am not sure if setting the `Stance` explicitly to `Msf::Exploit::Stance::Aggressive` is necessary. It seems to be the default here.